modules/nw-bgp-examples.adoc

```diff
@@ -0,0 +1,259 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp-routing/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-examples_{context}"]
+= Examples advertising pod IP addresses with BGP
+
+The following examples describe several different configurations for advertising pod IP addresses with BGP. The external network border router has the `172.18.0.5` IP address.
+
+Each example relies upon the following `FRRConfiguration` object, which OVN-Kubernetes generated from a `RouteAdvertisements` CR:
+
+.`FRRConfiguration` CR generated by OVN-Kubernetes
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  creationTimestamp: "2024-06-11T14:22:37Z"
+  generation: 1
+  name: metallb-ovn-worker
+  namespace: openshift-frr-k8s
+  resourceVersion: "1323"
+  uid: 99b64be3-4f36-4e0b-8704-75aa5182d89f
+  labels:
+    routeAdvertisements: default
+spec:
+  bgp:
+    routers:
+    - asn: 64512
+      neighbors:
+      - address: 172.18.0.5
+        asn: 64512
+        disableMP: false
+        holdTime: 1m30s
+        keepaliveTime: 30s
+        passwordSecret: {}
+        port: 179
+        toAdvertise:
+          allowed:
+            mode: filtered
+        toReceive:
+          allowed:
+            mode: filtered
+  nodeSelector:
+    matchLabels:
+       kubernetes.io/hostname: ovn-worker
+----
+
+The OVN-Kubernetes controller checks if this `FRRConfiguration` CR applies to its node as indicated by the `kubernetes.io/hostname: ovn-worker` label. For these examples a user-defined network named `blue` exists with a network of `10.0.0.0/16` and a matching VRF device exists in the Linux host. The slice of this supernet allocated to nodes with the `kubernetes.io/hostname: ovn-worker` label is `10.0.1.0/24`.
```

Comment:
`The OVN-Kubernetes controller checks if this `FRRConfiguration` CR applies to its node as indicated by the `kubernetes.io/hostname: ovn-worker` label.`

should really be

`The OVN-Kubernetes controller checks that the RouteAdvertisements selected nodes is a subset of the nodes selected by the RouteAdvertisements selected FRRConfigurations`